GVFS/GVFS.FunctionalTests/FileSystemRunners/BashRunner.cs

```diff
@@ -246,7 +254,14 @@ public override void DeleteFile_FileShouldNotBeFound(string path)
 
         public override void DeleteFile_AccessShouldBeDenied(string path)
         {
-            this.DeleteFile(path).ShouldContain(permissionDeniedMessage);

```

Comment:
I think I would rather change this to `this.DeleteFile(path); path.ShouldBeAFile();`. Regardless of what the error is, what we care about is that the file is still present. And that check is also platform-agnostic.